lab3/lab3_task_5.html

```diff
@@ -0,0 +1,289 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8"/>
+    <title>lab3_5</title>
+    <style>
+        /* form syle section*/
+        input#telephone {
+            width: 100px;
+        }
+        input + span {
+            padding-right: 30px;
+        }
+        input#telephone:invalid+span:after, 
+        input#email:invalid+span:after {
+            position: absolute; content: '✖';
+            padding-left: 5px;
+        }
+        input#telephone:valid+span:after, 
+        input#email:valid+span:after {
+            position: absolute;
+            content: '✓';
+            padding-left: 5px;
+        }
+        #telephone:valid {
+            display: inline-block;
+        }
+        input[id="email"]:valid ~ input#submit, input[id="tel"]:valid ~ input#submit  {
+            display: inline-block;            
+        }
+        #submit {
+            display: none;
+        }       
+        /* TABLE Style */
+        table {
+            border-spacing: 0;
+            border-collapse: collapse;
+            overflow: hidden;
+            z-index: 1;
+        }
+        tr:nth-child(odd) {  
+            background-color: rgba(13, 134, 122, .1)
+        }       
+        tr:nth-child(even) {  
+            background-color: rgba(222, 251, 248, .1);
+        }
+        td, th, .row, .col {
+            cursor: pointer;
+            padding: 10px;
+            position: relative;
+        }
+        td:hover::before,
+        .row:hover::before { 
+            background-color: #ffa;
+            content: '\00a0';  
+            height: 100%;
+            left: -5000px;
+            position: absolute;  
+            top: 0;
+            width: 10000px;   
+            z-index: -1;        
+        }
+        td:hover::after,
+        .col:hover::after { 
+            background-color: #ffa;
+            content: '\00a0';  
+            height: 10000px;    
+            left: 0;
+            position: absolute;  
+            top: -5000px;
+            width: 100%;
+            z-index: -1;        
+        }
+        /* activity selectors */
+        input[id="boxing"]:checked ~ table tr.boxing {               
+            display: none;           
+        }
+        input[id="yoga"]:checked ~ table tr.yoga {               
+            display: none;           
+        }
+        input[id="power_lifting"]:checked ~ table tr.power_lifting {               
```

Comment:
Either use hypen-separated-names or camelCaseNames for CSS IDs and Class Names. Do not use underscores.

Hypens are recommended by Google:
https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

Hypens are also used in BEM, which you're likely to use in the future:
https://en.bem.info/methodology/naming-convention/

camelCase can also be used if you're not using BEM or stick to Google conventions.